docs/reference/buildx_du.md

```diff
@@ -60,6 +60,30 @@ The asterisks (\*) in the output indicate the following:
   If you prune such a record then you will lose build cache but only metadata
   will be deleted as the image still needs to actual storage layers.
 
+### <a name="verbose"></a> Use verbose output (--verbose)
+
+The verbose output of the `docker buildx du` command is useful for inspecting
+the disk usage records in more detail. The verbose output shows the mutable and
+shared states more clearly, as well as additional information about the
+corresponding layer.
+
+```console
+$ docker buildx du --verbose
+ID:    6q7ikfbhueqke0fvsactbvska
+Parent:    z0mydj2w9mwv9jkotgq4vhpmx
+Created at:  2023-11-20 15:32:16.482204879 +0000 UTC
+Mutable:  true
+Reclaimable:  true
+Shared:    false
+Size:    36.86kB
+Description:  mount / from exec /bin/sh -c htmltest
+Usage count:  1
+Last used:  22 hours ago
+Type:    regular
+
+...
```

Comment:
Output doesn't seem aligned correctly:

```
...
Last used:      2 days ago
Type:           regular

ID:             05d0elirb4mmvpmnzbrp3ssrg
Parent:         e8sfdn4mygrg7msi9ak1dy6op
Created at:     2023-11-20 09:53:30.881558721 +0000 UTC
Mutable:        false
Reclaimable:    true
Shared:         false
Size:           0B
Description:    [gobase 3/3] WORKDIR /src
Usage count:    3
Last used:      24 hours ago
Type:           regular

Reclaimable:    4.453GB
Total:          4.453GB
```

Reply:
yea it's weird I got this misalignment in my terminal as well 🤔

Reply:
updated